atlp-rwanda/hackers-ec-Fe
# #187419008 User should be able to update their password

### What does this PR do?

#### Password Update

#### Description of Task to be completed?

clone `feat-user-update-password-187419008` branch and then do `npm install`

#### How should this be manually tested?

<ul>
    <li>make sure that you are logged in</li>
    <li>go to profile picture on top right corner and click it</li>
    <li>click on password and test the fiture</li>
</ul>

#### What are the relevant pivotal tracker/Trello stories?

#187419008
